.env.blockscoutstagingrc1

```diff
@@ -0,0 +1,519 @@
+# Don't use "//" for comments in this file.
+# This file is meant to be executed as a bash script for testing.
+ENV_TYPE="production"
+
+GETH_VERBOSITY=2
+GETH_ENABLE_METRICS=false
+
+VM_BASED=true
+
+KUBERNETES_CLUSTER_NAME="celo-networks-dev"
+KUBERNETES_CLUSTER_ZONE="us-west1-a"
+CLUSTER_DOMAIN_NAME="celo-testnet"
+
+TESTNET_PROJECT_NAME="celo-testnet"
+
+AZURE_KOMENCI_SOUTHBR_AZURE_SUBSCRIPTION_ID=7a6f5f20-bd43-4267-8c35-a734efca140c
```

Comment:
Since it's blockscout staging env, we could delete irrelevant values from the env file, anything that is related to oracles, forno, komenci, odis, walletconnect and so on is not needed.